Engine/include/math/VectorMath.hpp

```diff
@@ -0,0 +1,189 @@
+//
+// VectorMath.hpp
+// VGD Engine
+//
+// Created by Benjamin Arnold on 29 Aug 2015
+//
+
+/*! \file VectorMath.hpp
+* @brief defines common vector math operations.
+*/
+
+#pragma once
+
+namespace vgd {
```

Comment:
vgd is fine, we shouln't be using find replace anyways since it can get false positives. Instead we should use refactor tools such as visual assist.